components/formats-gpl/test/matlab/TestBfCheckJavaPath.m

```diff
@@ -78,5 +80,11 @@ function testPerformance(self)
             % bfCheckJavaPath should return fast
             assertTrue(mean(times(3:end)) < self.maxTime);
         end
+        
+        function testJavaMethod(self)
+            self.version = char(loci.formats.FormatTools.VERSION);
+            version = char(java_get('loci.formats.FormatTools', 'VERSION'));
+            assertEqual( self.version, version);
```

Comment:
Can we also test the signature of `bfCheckJavaPath()` where the version is returned as a second output is returned here, i.e.

```
[self.status, self.version] = bfCheckJavaPath(false);
```